src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -147,20 +147,17 @@ internal static SecurityZone GetFileSecurityZone(string filePath)
         {
```

Comment:
There are some other checks this code does, however, upon closer inspection, I think they also don't apply to Unix so I'll `#ifdef` this code out on UNIX